### ISSUE TITLE
just ... 

### ISSUE THREAD

**Issue by TLINDEN** [dthree/cash#11]
> No ugly DLLs
> Just:
> npm install cash -g
> cash
> $

not really:
```
C:\Users\ik00888>npm install cash -g
Der Befehl "npm" ist entweder falsch geschrieben oder
konnte nicht gefunden werden.
```
And of course in order to get npm running on a system, lots of "ugly dll's" have to be installed.

So, nice PR, but you guys can't just pretend that everyone has this stuff installed or even knows how to get there. That's ridiculous.

just my 0.02$,
Tom

**Comment by michaelowens** [comment]
That's strange, I never get German errors on my PC.

**Comment by TLINDEN** [comment]
Yeah, that's just an error message you get if something isn't installed :) Don't get me wrong: I know how to install node.js and the like. I found it just annoying that it is presumed that everyone reading the README knows this.

It's like: "How do I install node.js?" "Well, just get the latest ports from svn and install www/node". Here I presume everyone has of course FreeBSD installed and is familiar with installing ports. Ridiculous.


- Tom

**Comment by GijsGoudzwaard** [comment]
You can use the windows installer available on https://nodejs.org/

**Comment by bfred-it** [comment]
I see your point but `cash` is a `node` tool, it's built for who already has node installed.

Read that as _No ugly DLLs (other than `node`'s, which you already use)_

**Comment by TLINDEN** [comment]
> I see your point but cash is a node/ES6 tool,
> it's built for who already has node installed.

pretty clear. But I found it by looking for a cygwin alternative. I've nothing to do with node.js, but cash would be interesting nevertheless. I may not be part the intended audience, however I think it is a good thing "outsiders" want to use it, isn't it?
So, a mention of node.js in the README as @GijsGoudzwaard did, would made things more clear.

- Tom

**Comment by dthree** [comment]
Working on an MSI installer for to make things simpler for non-node users! Additionally I'll throw some links in with instructions on installing node.

**Comment by blockloop** [comment]
Is it plausible to suggest using [chocolatey](https://chocolatey.org/packages/nodejs) to install nodejs? Perhaps it can be listed as one installation option for those who already have chocolatey installed.

**Comment by dthree** [comment]
For sure! Chocolately is a great idea! Feel free to do a PR on the readme. And I'll do some research on making a package there.

**Comment by dthree** [comment]
@TLINDEN Haven't gotten a response from you in a bit and you weren't particularly suggesting anything constructive to solve the problem, so I changed the issue title. Feel free to chime in though on constructive suggestions that help make Cash more accessible, which I'm all for.

**Comment by TLINDEN** [comment]
@dthree: not sure what you're expecting from me. As I already explained I don't have any clue about node.js or its ecosystem. So, please don't expect a PR from me. Also, I don't think it's not in-constructive to point out documentation inconsistencies by clueless outsiders.

**Comment by dthree** [comment]
No probs - don't expect a pr. I was more trying to figure out what you suggested to solve the inconsistency, but I understand you now.
